dashboard-tw-wallet/src/lib/utils/transactionCalculations.ts

```
@@ -0,0 +1,63 @@
+export const calculateTotalEscrowedAmount = (totalFees: string) => {
+  // Extract the numeric part by removing non-numeric characters except decimal point
+  const numericPart = totalFees.replace(/[^\d.-]/g, '');
+  
+  // Handle empty or invalid numeric strings
+  if (!numericPart || numericPart === '-') {
+    return '0.00 XLM';
+  }
+
+  // Parse the numeric value with validation
+  const totalFeesNum = parseFloat(numericPart);
+  if (isNaN(totalFeesNum) || !isFinite(totalFeesNum)) {
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)